src/content/self-hosted/helm-configuration.mdx

```diff
@@ -493,6 +493,22 @@ globals:
       effect: "NoSchedule"
 ```
 
+- `nodeSelectors`: Node selectors to be added to Pods that are either under the `okteto` or `dev` categories, as shown below. This helps you match Pods to nodes whose labels match the node selecotr.
+  - `okteto`: A dictionary of node selectors to be added to Pods related to Okteto components (i.e. Okteto api), which run in the `okteto` namespace. All the node selectors specified here will be applied in conjunction with the single node selector created by [`tolerations.oktetoPool`](self-hosted/helm-configuration.mdx#tolerations) until deprecation of `tolerations.oktetoPool`.
+  - `dev`: A dictionary of node selectors to be added to Pods created by user applications, which run in namespaces managed by Okteto. All the node selectors specified here will be applied in conjunction with the single node selector created by [`tolerations.devPool`](self-hosted/helm-configuration.mdx#tolerations) until deprecation of `tolerations.devPool`.
+
+```yaml
+globals:
+  nodeSelectors:
+    okteto:
+      oktetoSeletorOne: valueOne
```

Comment:
Could we use a more realistic example? For example, to select nodes with `okteto-node-label: okteto` and ``okteto-node-label: dev`